src/Inceptum.AppServer/AppDiscovery/NuGet/NugetApplicationRepository.cs

```diff
@@ -139,5 +145,10 @@ private void cleanUpInstallFolder(string installPath)
                 File.Delete(packagesConfig);
             }
         }
+
+        private static void ConfigureSupportedSslProtocols()
+        {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12;

```

Comment:
Why do you need it?
Looks like attempt to change appserver code to adopt for obsolete or misconfigured environment...